D/Completion Rules.tmPreferences

```diff
@@ -6,7 +6,7 @@
     <key>settings</key>
     <dict>
         <key>cancelCompletion</key>
-        <string>^\s*(\}?\s*(else|try|do|finally|synchonized|struct|union|unittest|debug)|(class|struct|interface|union)\s*[a-zA-Z_0-9]+*)$</string>
+        <string>^\s*(\}?\s*(else|try|do|finally|synchronized|struct|union|unittest|debug)|(class|struct|interface|union)\s*[a-zA-Z_0-9]+*)$</string>
```

Comment:
notable

Reply:
Fixes like that one really would also fix functional bugs. Nice findings.

Reply:
This, yes, is very important.